sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_models.py

```diff
@@ -154,9 +158,10 @@ class FeatureFlagConfigurationSetting(
     )
     kind = "FeatureFlag"
 
-    def __init__(self, feature_id, enabled, filters=[], **kwargs):  # pylint: disable=dangerous-default-value
+    def __init__(self, feature_id, filters=[], **kwargs):  # pylint: disable=dangerous-default-value
```

Comment:
Curious, why we move enabled into kwargs but not filters as well?

Reply:
Will move both in